pkg/internal/client/client.go

```diff
@@ -21,16 +21,22 @@ package client
 
 import (
 	"context"
+	"log"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8scorev1 "k8s.io/api/core/v1"
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
nit: These were previously defined as "corev1" and "metav1".

Reply:
yes I changed it to be be more unified. After this PR I'll recheck if I missed anything and unify the entire repo.

Reply:
I'll return it and change it in another PR